### ISSUE TITLE
[Issue]: address already in use

### ISSUE THREAD

**Issue by libook** [jellyfin/jellyfin#11618]
### Please describe your bug

It was working fine before upgrading to 10.9.0. After being automatically upgraded to 10.9.0 by Watchtower it won't start.

### Reproduction Steps

OS: unRAID
Docker container network in **host** mode.

network.xml
```
<?xml version="1.0" encoding="utf-8"?>
<NetworkConfiguration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <BaseUrl />
  <EnableHttps>false</EnableHttps>
  <RequireHttps>false</RequireHttps>
  <CertificatePath />
  <CertificatePassword />
  <InternalHttpPort>8096</InternalHttpPort>
  <InternalHttpsPort>8920</InternalHttpsPort>
  <PublicHttpPort>8096</PublicHttpPort>
  <PublicHttpsPort>8920</PublicHttpsPort>
  <AutoDiscovery>true</AutoDiscovery>
  <EnableUPnP>true</EnableUPnP>
  <EnableIPv4>true</EnableIPv4>
  <EnableIPv6>false</EnableIPv6>
  <EnableRemoteAccess>true</EnableRemoteAccess>
  <LocalNetworkSubnets />
  <LocalNetworkAddresses />
  <KnownProxies />
  <IgnoreVirtualInterfaces>true</IgnoreVirtualInterfaces>
  <VirtualInterfaceNames>
    <string>veth</string>
  </VirtualInterfaceNames>
  <EnablePublishedServerUriByRequest>false</EnablePublishedServerUriByRequest>
  <PublishedServerUriBySubnet />
  <RemoteIPFilter />
  <IsRemoteIPFilterBlacklist>false</IsRemoteIPFilterBlacklist>
</NetworkConfiguration>
```

None returned by `netstat -tulpn | grep 8096`. So no other process occupies this port.

### Jellyfin Version

10.9.0

### Environment

```markdown
- OS: unRAID 6.12.10
- Linux Kernel: 6.1.79-Unraid
- Virtualization: Docker
- Clients: Browser
- Browser: Firefox 125.0.1
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- GPU Model:
- Plugins: 
  - Anime 11.2012.600.2703
  - Cover Art Archive 7.0.0.0
  - Open Subtitles 19.0.0.0
  - Skin Manager 2.0.1
  - TMDb 10.9.1.0
  - Studio Images 10.9.1.0
  - OMDb 10.9.1.0
  - MusicBrainz 10.9.1.0
  - AudioDB 10.9.1.0
- Reverse Proxy: nginx
- Base URL: none
- Networking: Host
- Storage: local
```

### Jellyfin logs

```shell
[10:03:44] [INF] [1] Main: Jellyfin version: 10.9.1
[10:03:44] [INF] [1] Main: Environment Variables: ["[JELLYFIN_WEB_DIR, /jellyfin/jellyfin-web]", "[JELLYFIN_CACHE_DIR, /cache]", "[JELLYFIN_CONFIG_DIR, /config/config]", "[JELLYFIN_FFMPEG, /usr/lib/jellyfin-ffmpeg/ffmpeg]", "[JELLYFIN_DATA_DIR, /config]", "[JELLYFIN_LOG_DIR, /config/log]"]
[10:03:44] [INF] [1] Main: Arguments: ["/jellyfin/jellyfin.dll", "--ffmpeg", "/usr/lib/jellyfin-ffmpeg/ffmpeg"]
[10:03:44] [INF] [1] Main: Operating system: Debian GNU/Linux 12 (bookworm)
[10:03:44] [INF] [1] Main: Architecture: X64
[10:03:44] [INF] [1] Main: 64-Bit Process: True
[10:03:44] [INF] [1] Main: User Interactive: True
[10:03:44] [INF] [1] Main: Processor count: 16
[10:03:44] [INF] [1] Main: Program data path: /config
[10:03:44] [INF] [1] Main: Log directory path: /config/log
[10:03:44] [INF] [1] Main: Config directory path: /config/config
[10:03:44] [INF] [1] Main: Cache path: /cache
[10:03:44] [INF] [1] Main: Web resources path: /jellyfin/jellyfin-web
[10:03:44] [INF] [1] Main: Application directory: /jellyfin/
[10:03:44] [INF] [1] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: /cache
[10:03:44] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loading assemblies
[10:03:44] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded assembly Jellyfin.Plugin.Anime, Version=11.2012.600.2703, Culture=neutral, PublicKeyToken=null from /config/plugins/Anime_11.2012.600.2703/Jellyfin.Plugin.Anime.dll
[10:03:44] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Skipping disabled plugin 9.0.0.0 of Bookshelf 
[10:03:44] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Skipping disabled plugin 10.0.0.0 of Bookshelf 
[10:03:44] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded assembly Jellyfin.Plugin.CoverArtArchive, Version=7.0.0.0, Culture=neutral, PublicKeyToken=null from /config/plugins/Cover Art Archive_7.0.0.0/Jellyfin.Plugin.CoverArtArchive.dll
[10:03:44] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded assembly MetaBrainz.Common.Json, Version=5.0.0.0, Culture=neutral, PublicKeyToken=be12cbb629c03021 from /config/plugins/Cover Art Archive_7.0.0.0/MetaBrainz.Common.Json.dll
[10:03:44] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded assembly MetaBrainz.MusicBrainz.CoverArt, Version=4.0.0.0, Culture=neutral, PublicKeyToken=be12cbb629c03021 from /config/plugins/Cover Art Archive_7.0.0.0/MetaBrainz.MusicBrainz.CoverArt.dll
[10:03:44] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Skipping disabled plugin 1.0.7.0 of Douban 
[10:03:44] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded assembly Jellyfin.Plugin.OpenSubtitles, Version=19.0.0.0, Culture=neutral, PublicKeyToken=null from /config/plugins/Open Subtitles_19.0.0.0/Jellyfin.Plugin.OpenSubtitles.dll
[10:03:44] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded assembly Jellyfin.Plugin.SkinManager, Version=1.5.0.0, Culture=neutral, PublicKeyToken=null from /config/plugins/Skin Manager_2.0.1/Jellyfin.Plugin.SkinManager.dll
[10:03:44] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Skipping disabled plugin 9.0.0.0 of TheTVDB 
[10:03:44] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Skipping disabled plugin 10.0.0.0 of TheTVDB 
[10:03:44] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Skipping disabled plugin 11.0.0.0 of Webhook 
[10:03:44] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Defined LAN addresses: ["127.0.0.1/8", "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]
[10:03:44] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Defined LAN exclusions: []
[10:03:44] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using LAN addresses: ["127.0.0.1/8", "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]
[10:03:44] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using bind addresses: ["127.0.0.1", "192.168.50.49", "172.27.0.1", "172.26.0.1", "172.20.0.1", "172.22.0.1", "192.168.96.1", "172.18.0.1", "172.17.0.1", "172.19.0.1", "172.23.0.1", "172.21.0.1", "192.168.50.49"]
[10:03:44] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Remote IP filter is Allowlist
[10:03:44] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Filter list: []
[10:03:50] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: Anime 11.2012.600.2703
[10:03:50] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: Cover Art Archive 7.0.0.0
[10:03:50] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: Open Subtitles 19.0.0.0
[10:03:50] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: Skin Manager 2.0.1
[10:03:50] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: TMDb 10.9.1.0
[10:03:50] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: Studio Images 10.9.1.0
[10:03:50] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: OMDb 10.9.1.0
[10:03:50] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: MusicBrainz 10.9.1.0
[10:03:50] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: AudioDB 10.9.1.0
[10:03:50] [WRN] [1] Microsoft.AspNetCore.DataProtection.Repositories.EphemeralXmlRepository: Using an in-memory repository. Keys will not be persisted to storage.
[10:03:50] [WRN] [1] Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager: Neither user profile nor HKLM registry available. Using an ephemeral key repository. Protected data will be unavailable when application exits.
[10:03:50] [INF] [1] Main: Kestrel is listening on 127.0.0.1
[10:03:50] [INF] [1] Main: Kestrel is listening on 192.168.50.49
[10:03:50] [INF] [1] Main: Kestrel is listening on 172.27.0.1
[10:03:50] [INF] [1] Main: Kestrel is listening on 172.26.0.1
[10:03:50] [INF] [1] Main: Kestrel is listening on 172.20.0.1
[10:03:50] [INF] [1] Main: Kestrel is listening on 172.22.0.1
[10:03:50] [INF] [1] Main: Kestrel is listening on 192.168.96.1
[10:03:50] [INF] [1] Main: Kestrel is listening on 172.18.0.1
[10:03:50] [INF] [1] Main: Kestrel is listening on 172.17.0.1
[10:03:50] [INF] [1] Main: Kestrel is listening on 172.19.0.1
[10:03:50] [INF] [1] Main: Kestrel is listening on 172.23.0.1
[10:03:50] [INF] [1] Main: Kestrel is listening on 172.21.0.1
[10:03:50] [INF] [1] Main: Kestrel is listening on 192.168.50.49
[10:03:50] [WRN] [1] Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager: No XML encryptor configured. Key {f404d974-9c8b-4b65-a1c1-a493514dc5a3} may be persisted to storage in unencrypted form.
[10:03:51] [INF] [1] Jellyfin.Networking.PortForwardingHost: Starting NAT discovery
[10:03:51] [WRN] [1] Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware: The WebRootPath was not found: /wwwroot. Static files may be unavailable.
[10:03:51] [ERR] [1] Microsoft.Extensions.Hosting.Internal.Host: Hosting failed to start
System.IO.IOException: Failed to bind to address http://192.168.50.49:8096: address already in use.
 ---> Microsoft.AspNetCore.Connections.AddressInUseException: Address already in use
 ---> System.Net.Sockets.SocketException (98): Address already in use
   at System.Net.Sockets.Socket.DoBind(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.Sockets.Socket.Bind(EndPoint localEP)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketTransportOptions.CreateDefaultBoundListenSocket(EndPoint endpoint)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketConnectionListener.Bind()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketConnectionListener.Bind()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketTransportFactory.BindAsync(EndPoint endpoint, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure.TransportManager.BindAsync(EndPoint endPoint, ConnectionDelegate connectionDelegate, EndpointConfig endpointConfig, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.<>c__DisplayClass28_0`1.<<StartAsync>g__OnBind|0>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindEndpointAsync(ListenOptions endpoint, AddressBindContext context, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindEndpointAsync(ListenOptions endpoint, AddressBindContext context, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.ListenOptions.BindAsync(AddressBindContext context, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.EndpointsStrategy.BindAsync(AddressBindContext context, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindAsync(ListenOptions[] listenOptions, AddressBindContext context, Func`2 useHttps, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.BindAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.StartAsync[TContext](IHttpApplication`1 application, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.Internal.Host.<StartAsync>b__15_1(IHostedService service, CancellationToken token)
   at Microsoft.Extensions.Hosting.Internal.Host.ForeachService[T](IEnumerable`1 services, CancellationToken token, Boolean concurrent, Boolean abortOnFirstException, List`1 exceptions, Func`3 operation)
[10:03:51] [ERR] [1] Main: Kestrel failed to start! This is most likely due to an invalid address or port bind - correct your bind configuration in network.xml and try again
[10:03:51] [FTL] [1] Main: Error while starting server
System.IO.IOException: Failed to bind to address http://192.168.50.49:8096: address already in use.
 ---> Microsoft.AspNetCore.Connections.AddressInUseException: Address already in use
 ---> System.Net.Sockets.SocketException (98): Address already in use
   at System.Net.Sockets.Socket.DoBind(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.Sockets.Socket.Bind(EndPoint localEP)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketTransportOptions.CreateDefaultBoundListenSocket(EndPoint endpoint)
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketConnectionListener.Bind()
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketConnectionListener.Bind()
   at Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets.SocketTransportFactory.BindAsync(EndPoint endpoint, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure.TransportManager.BindAsync(EndPoint endPoint, ConnectionDelegate connectionDelegate, EndpointConfig endpointConfig, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.<>c__DisplayClass28_0`1.<<StartAsync>g__OnBind|0>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindEndpointAsync(ListenOptions endpoint, AddressBindContext context, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindEndpointAsync(ListenOptions endpoint, AddressBindContext context, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.ListenOptions.BindAsync(AddressBindContext context, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.EndpointsStrategy.BindAsync(AddressBindContext context, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.AddressBinder.BindAsync(ListenOptions[] listenOptions, AddressBindContext context, Func`2 useHttps, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.BindAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerImpl.StartAsync[TContext](IHttpApplication`1 application, CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Hosting.GenericWebHostService.StartAsync(CancellationToken cancellationToken)
   at Microsoft.Extensions.Hosting.Internal.Host.<StartAsync>b__15_1(IHostedService service, CancellationToken token)
   at Microsoft.Extensions.Hosting.Internal.Host.ForeachService[T](IEnumerable`1 services, CancellationToken token, Boolean concurrent, Boolean abortOnFirstException, List`1 exceptions, Func`3 operation)
   at Microsoft.Extensions.Hosting.Internal.Host.StartAsync(CancellationToken cancellationToken)
   at Jellyfin.Server.Program.StartServer(IServerApplicationPaths appPaths, StartupOptions options, IConfiguration startupConfig)
[10:03:51] [INF] [1] Main: Running query planner optimizations in the database... This might take a while
[10:03:51] [INF] [1] Emby.Server.Implementations.ApplicationHost: Disposing CoreAppHost
[10:03:51] [INF] [1] Emby.Server.Implementations.ApplicationHost: Disposing MusicBrainzArtistProvider
[10:03:51] [INF] [1] Emby.Server.Implementations.ApplicationHost: Disposing MusicBrainzAlbumProvider
[10:03:51] [INF] [1] Emby.Server.Implementations.ApplicationHost: Disposing CoverArtArchiveImageProvider
[10:03:51] [INF] [1] Emby.Server.Implementations.ApplicationHost: Disposing PluginManager
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gnattu** [comment]
See #11587

Should be fixed in 10.9.2

**Comment by gnattu** [comment]
Duplicate of #11557
